podvm-payload/Dockerfile

```diff
@@ -73,6 +73,16 @@ WORKDIR /workdir/guest-components/confidential-data-hub/hub
 RUN cargo build --verbose --release --no-default-features --features "kbs,bin,ttrpc"
 RUN cp /workdir/guest-components/target/release/ttrpc-cdh /artifacts/usr/local/bin/confidential-data-hub
 
+# build k8s pause image
+FROM registry.k8s.io/pause:3.9 as pause_binary
+FROM registry.access.redhat.com/ubi9/ubi:latest as pause_builder
+RUN dnf install -y runc jq
+COPY --from=pause_binary /pause /pause_bundle/rootfs/
+RUN runc spec --rootless && \
+    jq '.process.args = ["/pause"] | .root.path = "rootfs"' config.json > /pause_bundle/config.json && \
+    rm -f config.json
+RUN tar czvf /pause-bundle.tar.gz -C / pause_bundle/
+
```

Comment:
@snir911 I think changes should also be made here https://github.com/openshift/sandboxed-containers-operator/blob/8b340c9e010a0cb8eff5e3acd19aa39d51bc9e79/config/peerpods/podvm/lib.sh#L11-L12
Otherwise I think during image creation the bundle might be replaced.

Reply:
right, I have the change ready, however, assuming the packer builds should be deprecated, it may be better leave it as is, and adopt the pause usage by bootc & precreated images.

Reply:
Even for packer it can be replaced as kata agent takes care of executing the correct pause binary based on embedded image.  Having said that it's not necessary to make packer related changes as anyway we should move away from it given pre-built images are available.